examples/SessionMainWindow_example.py

```diff
@@ -26,6 +26,12 @@ def __init__(self, *args, **kwargs):
         add_every_widget_to_form(form_widget)
         self.setCentralWidget(form_widget)
 
+    def addToMenu(self):
+        ''' Adds an example extra menu.'''
+        extra_menu = self.menu_bar.addMenu("Extra Menu")
+        self.menu_bar.addMenu(extra_menu)
```

Comment:
I was very confused by these lines, however I see that the confusion comes from Qt
https://doc.qt.io/qt-6/qmenubar.html#public-functions
`addMenu` called with a string returns a `QMenu`, if called with a `QMenu` returns a `QAction`